lib/npg_tracking/data/transcriptome/find.pm

```diff
@@ -62,21 +117,9 @@ has 'rnaseqc_gtf_file' => ( isa           => q{Maybe[Str]},
 
 sub _build_rnaseqc_gtf_file {
   my $self = shift;
-  my @gtf_files;
-  if ($self->rnaseqc_gtf_path) {
-    @gtf_files = glob $self->rnaseqc_gtf_path . '/*.gtf';
-  }
-  if (scalar @gtf_files > 1) {
-    croak 'More than 1 gtf file in ' . $self->rnaseqc_gtf_path;
-  }
-
-  if (scalar @gtf_files == 0) {
-    if ($self->_organism_dir && -d $self->_organism_dir) {
-      $self->messages->push('Directory ' . $self->_organism_dir . ' exists, but GTF file not found');
-    }
-    return;
-  }
-  return $gtf_files[0];
+  $self->_set_subfolder('RNA-SeQC');
+  $self->_set_file_type('gtf');
+  return $self->_find_file;
```

Comment:
Reapeteadly resetting values for attributes _file_type and _subfolder might have unexpected side effects. It would be cleaner for the '_find_file' method to accept file type and subfolder as input.